packaging/linux/build.spec

```diff
@@ -51,18 +50,48 @@ sudo pip wheel git+https://%{GITHUB_USERNAME}:%{GITHUB_PASSWORD}@github.com/clou
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-openstack-plugin/%{PLUGINS_TAG_NAME}/plugin.yaml -o %{buildroot}/cfy/cloudify/plugins/openstack-plugin/plugin.yaml &&
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-diamond-plugin/%{PLUGINS_TAG_NAME}/plugin.yaml -o %{buildroot}/cfy/cloudify/plugins/diamond-plugin/plugin.yaml &&
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-aws-plugin/%{PLUGINS_TAG_NAME}/plugin.yaml -o %{buildroot}/cfy/cloudify/plugins/aws-plugin/plugin.yaml &&
+
+# Clone and copy commercial plugin.yaml files to local plugins folder
+git clone https://%{GITHUB_USERNAME}:%{GITHUB_PASSWORD}@github.com/cloudify-cosmo/cloudify-vsphere-plugin.git /tmp/cloudify-vsphere-plugin
+cp /tmp/cloudify-vsphere-plugin/plugin.yaml %{buildroot}/cfy/cloudify/plugins/vsphere-plugin &&
+git clone https://%{GITHUB_USERNAME}:%{GITHUB_PASSWORD}@github.com/cloudify-cosmo/cloudify-softlayer-plugin.git /tmp/cloudify-softlayer-plugin
+cp /tmp/cloudify-softlayer-plugin/plugin.yaml %{buildroot}/cfy/cloudify/plugins/softlayer-plugin &&
+
+# Download types.yaml
+sudo curl http://getcloudify.org.s3.amazonaws.com/spec/cloudify/%{CORE_TAG_NAME}/types.yaml -o %{buildroot}/cfy/cloudify/types/types.yaml &&
+
+# Download scripts
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/fs/mkfs.sh -o %{buildroot}/cfy/cloudify/scripts/mkfs.sh
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/fs/fdisk.sh -o %{buildroot}/cfy/cloudify/scripts/fdisk.sh
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/fs/mount.sh -o %{buildroot}/cfy/cloudify/scripts/mount.sh
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/fs/unmount.sh -o %{buildroot}/cfy/cloudify/scripts/unmount.sh
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/policies/host_failure.clj -o %{buildroot}/cfy/cloudify/scripts/host_failure.clj
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/policies/threshold.clj -o %{buildroot}/cfy/cloudify/scripts/threshold.clj
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/policies/ewma_stabilized.clj -o %{buildroot}/cfy/cloudify/scripts/ewma_stabilized.clj
+sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/rest-service/cloudify/triggers/execute_workflow.clj -o %{buildroot}/cfy/cloudify/scripts/execute_workflow.clj
 
```

Comment:
sudo curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/%{CORE_TAG_NAME}//resources/res
Please remove the unnecessary '/' and test it